tests/test_vaccine_design.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import tempfile
+import py_compile
+import subprocess
+from filecmp import cmp
+import os
+import sys
+
+#python -m unittest tests/test_vaccine_design.py
+class TestVaccineDesign(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        base_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
```

Comment:
`os.path.dirname(path)` is preferable to `os.path.join(path, '..')` mostly for consistency's sake